confignode/src/main/java/org/apache/iotdb/confignode/manager/load/LoadManager.java

```diff
@@ -69,8 +73,8 @@ public class LoadManager {
 
   private final long heartbeatInterval =
       ConfigNodeDescriptor.getInstance().getConf().getHeartbeatInterval();
```

Comment:
need add a method, which remove a node from DataNodeHeartbeatCache.  because we have operation of removing node.

Reply:
fixed~